.github/workflows/ci.yml

```diff
@@ -85,13 +85,16 @@ jobs:
     - name: Install Dependencies
       run: poetry install
       shell: bash
+    - name: Configure Git
+        run: |
+          git config user.name "github-actions"
+          git config user.email "actions@users.noreply.github.com"
+          git remote set-url origin https://${{ secrets.GH_TOKEN }}@github.com/<your-username>/<your-repo>.git
```

Comment:
I think the url still have some place holders here

Reply:
oops yes. Great catch

Reply:
Fixed!